### ISSUE TITLE
Getting API time out exceeded

### ISSUE THREAD

**Issue by tbone7467** [derek-miller/homebridge-unifi-smartpower#38]
**Describe Your Problem:**

**Logs:**

```
Show the Homebridge logs here, remove any sensitive information.
```

**Plugin Config:**

```json
{
    "name": "UniFi SmartPower",
    "host": "10.0.1.1",
    "port": 443,
    "username": "12345",
    "password": "12345",
    "controlSwitch": {
        "create": true,
        "name": "UniFi Control Enabled",
        "timeout": 60,
        "guardOutlets": true,
        "guardSwitchPorts": true
    },
    "excludeOutlets": [
        "70A741923715.1",
        "70A741923715.2",
        "70A741923715.3",
        "70A741923715.4",
        "70A741923715.5",
        "70A741923715.6",
        "70A741923715.7",
        "70A741923715.8",
        "70A741923715.9",
        "70A741923715.11",
        "70A741923715.12",
        "70A741923715.13",
        "70A741923715.14",
        "70A741923715.15",
        "70A741923715.17",
        "70A741923715.18"
    ],
    "includePorts": [
        "F492BF9E144C.6"
    ],
    "excludePorts": [
        "E438837A6D1F.4",
        "E438837A6D1F.5",
        "E438837A6D1F.6",
        "E438837A6D1F.7",
        "E438837A6D1F.8"
    ],
    "includeInactivePorts": false,
    "refreshDevicesPollInterval": 600,
    "outletStatusCacheTtl": 15,
    "outletStatusPollInterval": 15,
    "logApiResponses": false,
    "_bridge": {
        "username": "0E:7E:D9:0E:9E:5D",
        "port": 32373
    },
    "platform": "UniFiSmartPower"
}
```

**Screenshots:**

**Environment:**

- **Plugin Version**: 2.1.1
- **Homebridge Version**: <!1..8.2->
- **Node.js Version**: <!20.14.0->
- **NPM Version**: 
- **Operating System**: <!-Raspberry pi>

**Comment by derek-miller** [comment]
There are no known issues with the plugin and I am able to connect just fine. Timeouts are most likely related to incorrect configuration or firewall rules. Double check your configuration for any mistakes.

**Comment by tbone7467** [comment]
Thank you. I appreciate the guidance. Sent from my iPhoneOn Jun 5, 2024, at 10:34 AM, Derek Miller ***@***.***> wrote:﻿
Closed #38 as completed.
